telegram/_telegramobject.py

```diff
@@ -327,6 +341,34 @@ def to_dict(self, recursive: bool = True) -> JSONDict:
             :obj:`dict`
         """
         out = self._get_attrs(recursive=recursive)
+
+        # Now we should convert TGObjects to dicts inside objects such as sequences, and convert
+        # datetimes to timestamps. This mostly eliminates the need for subclasses to override
+        # `to_dict`
+        for key, value in out.items():
+            if isinstance(value, Sequence) and not isinstance(value, str):
```

Comment:
```suggestion
            if isinstance(value, Sequence) and not isinstance(value, (str, bytes)):
```

Strictly speaking the ` not isintance`  is not even necessary, as the `hasattr(item, "to_dict"` check will fail, but it's good to keep it so that we don't unnecessarily loop over things that we know are not TOs :D